src/utils/getHistogramChartConfig.js

```diff
@@ -19,6 +19,9 @@ such restriction.
 */
 import { round } from 'lodash'
 
+export const CHART_TYPE_LINE = 'line'
+export const CHART_TYPE_BAR = 'bar'
+
```

Comment:
Move this values back to `constants.js`. I meant to use the `CHART_TYPE_BAR` in the `getHistogramChartConfig`  because it also has type `bar`